app/src/main/java/com/github/kiolk/cowsandbulls/ui/screens/rules/RulesFragment.java

```diff
@@ -30,14 +30,24 @@ public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup c
         TextView mRules = view.findViewById(R.id.tv_rules_description);
         Resources resources = view.getContext().getResources();
         String text = resources.getString(R.string.rules);
+        String coloredText = "";
+        String bottomText = "";
+        StartSpannable clickableSpan = new StartSpannable((StartSpannable.StartSpannableOnclickListener) getActivity(),
+                getContext().getResources().getColor(R.color.middleBlue));
+
+        if (clickableSpan.getOnclickListener().gameIsStarted()) {
```

Comment:
Sorry, I didn't clear explain. Need to add only logic for close rule fragment, not change word start to close. Please roll back changes depend on with it.